files/docker-container-logrotate

```diff
@@ -1,9 +1,11 @@
 /var/lib/docker/containers/*/*.log {
   rotate 0
-  daily
+  hourly
```

Comment:
Just a question on understanding the usage of kube-dns. How much logs do we see in one-hour?